src/Missings.jl

```diff
@@ -514,4 +514,77 @@ julia> emptymissing(first)([1], 2)
 """
 emptymissing(f) = (x, args...; kwargs...) -> isempty(x) ? missing : f(x, args...; kwargs...)
 
+# Only for internal use. Allows dispatch over anonymous functions.
+struct MissingSmallest{T}
+    lt::T
+end
+
+"""
+    missingsmallest(f)
+
+Creates a function of two arguments `x` and `y` that tests whether `x` is less
+than `y` such that `missing` is always less than the other argument. In other 
+words, returns a modified version of the partial order function `f` such that
+`missing` is the smallest possible value, and all other non-`missing` values are
+compared according to `f`.
+
+# Examples
+```
+julia> missingsmallest((x, y) -> isless(x, 10*y))(missing, Inf)
+true
+
+julia> missingsless(-Inf, missing)
```

Comment:
this is an example of other definition

Reply:
Sorry, I think I dont' understand the comment

Reply:
This is docstring of `missingsmallest(f)` signature (see the top of the string), but you are showing example of `missingsless(-Inf, missing)` which is a signature taking two positional arguments.